piccolo/table.py

```diff
@@ -379,7 +379,10 @@ def __init__(
 
     @classmethod
     def _create_serial_primary_key(cls) -> Serial:
-        pk = Serial(index=False, primary_key=True, db_column_name="id")
+        if cls._meta.db.engine_type == "cockroach":
+            pk = Serial(index=False, primary_key=True, db_column_name="id", default="unique_rowid()") # Not sure where else to put "unique_rowid()". Has to be at table creation time.
```

Comment:
We need to refactor this slightly.

It's problematic when installing piccolo in a fresh project - it tries to find `piccolo_conf` and fails.

Maybe we can change this instead:

https://github.com/piccolo-orm/piccolo/blob/7754498705540c81c40ec203a7bb429d59a07da5/piccolo/columns/column_types.py#L777-L783

If we add this, hopefully it still works:

```python
        elif engine_type == "cockroachdb":
            return 'unique_rowid()'
```

Reply:
@dantownsend thank you for the invaluable feedback!

So while this works, I am concerned about the repercussions of using `unique_rowid()` instead of `DEFAULT` in piccolo operations after table creation time.

Cockroach should not care either way, though. I'm going to say while this is a low concern, should be fine.

Reply:
I've gone ahead and made the changes, updated the cockroach tests. All green as expected.